tiling.py

```diff
@@ -310,6 +315,119 @@ def fundamental_domain_transforms(self):
     def center(self):
         return self._center
 
+    @property
+    def diameter(self):
+        return self._diameter
+
+    @property
+    def radius(self):
+        return self._radius
+
+    @property
+    def group(self):
+        return self._group
+
+class vSquareKernel(object):
```

Comment:
Let's not copy-paste classes to make small modifications. Why not just simply augment the existing square kernel class to handle 3D cases. It should be 5 lines of code to do this...